internal/data/release_storage.go

```diff
@@ -0,0 +1,96 @@
+package data
+
+import (
+	"database/sql"
+	"time"
+
+	"github.com/ScienceSoft-Inc/integrity-sum/pkg/k8s"
+	"github.com/sirupsen/logrus"
+)
+
+//go:generate mockgen -source=release_storage.go -destination=mocks/mock_release_storage.go
```

Comment:
Not sure, do we need it?

Reply:
removed https://github.com/ScienceSoft-Inc/integrity-sum/pull/48/commits/bd9d6c5ff362a3286b340bc959ce34e96eb90ab5